elpy.el

```diff
@@ -537,6 +539,8 @@ virtualenv.
   :lighter " Elpy"
   (when (not (derived-mode-p 'python-mode))
     (error "Elpy only works with `python-mode'"))
+  (when (featurep 'xref)
+    (add-hook 'xref-backend-functions #'elpy--xref-backend nil t))
```

Comment:
Maybe use `(when (boundp 'xref-backend-functions) ...)` here? It's usually better to test for the functionality used instead of the feature.